DOCUMENTATION.md

```diff
@@ -49,66 +49,66 @@ client.close()
 The callback functions are functions which are executed when a trigger event happens.
 
 #### set_uplink_callback
-Set a callback function, to be called when an uplink message is received. It's possible to add more than one callback function.
+Add a callback function, to be called when an uplink message is received.
 ```python
 client.set_uplink_callback(uplink_callback)
 ```
 
 ##### uplink_callback
 The callback function must be declared in the script following this structure:
 * `uplink_callback(msg, client)`
-  * `msg`: **JSON**  the message received by the client
-  * `client`: **object**  the client from which the callback is executed are calling
+  * `msg`: **object**  the message received by the client.
+  * `client`: **object**  the client from which the callback is executed.
 
 #### set_connect_callback
-Set a connection callback function to be executed when the client connects to the broker. It's possible to add more than one callback function.
+Add a connection callback function to be executed when the client connects to the broker.
 ```python
 client.set_connect_callback(connect_callback)
 ```
 ##### connect_callback
 - `connect_callback(res, client)`: the function which will be executed on connection to the broker.
-  - `res`: **boolean**  the result of the connection. If it's true, the connection succeeded went well. If not, it means the connection failed.
+  - `res`: **boolean**  the result of the connection. If it's true, the connection succeeded. If not, it means the connection failed.
   - `client`: **object**  the TTN client from which the callback is called.
```

Comment:
Please clarify whether it is an `Object` (and in that class, the class of this object) or if it is a `dict`